.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug report
+description: Create a report if you believe something is not working
+title: "[BUG]: "
+labels: ["bug"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: How to reproduce the bug
+      description: Tell us how one can reproduce the bug
+      placeholder: I ran a demo or test that failed. The name of the demo is `demo_something.py`
+    validations:
+      required: true
+  - type: textarea
+    id: mwe-python
+    attributes:
+      label: Minimal Example (Python)
+      description: Add (optionally) a minimal script that reproduces the bug
+      render:
+      placeholder: |
+          import dolfinx
+          from mpi4py import MPI
+          domain = dolfinx.mesh.UnitSquareMesh(MPI.COMM_WORLD, 10, 10)
+    validations:
+      required: false
+
+  - type: textarea
+    id: output-python
+    attributes:
+      label: Output (Python)
+      description: If you get an error message or any output, please add it here
+      render: bash
+      placeholder: |
+          Traceback (most recent call last):
+          File "/root/shared/dolfinx/python/test/mwe.py", line 3, in <module>
+          domain = dolfinx.mesh.UnitSquareMesh(MPI.COMM_WORLD, 10, 10)
+          AttributeError: module 'dolfinx.mesh' has no attribute 'UnitSquareMesh'
+    validations:
+      required: false
+  
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 0.4.0
+        - 0.4.1
+        - 0.5.0
+        - 0.5.1
+        - main branch
```

Comment:
It is a pity this has to be specified manually.

Reply:
We could just make it a required text field instead of a Dropdown menu. However, I figured it is easier to have a limited number of options. We could in time deprecated minor versions.